docs/data-access/daco/renew-and-close.md

```diff
@@ -5,52 +5,64 @@ sidebar_label: Attesting, Renewing, & Closing
 
 ### How to Renew an Application
 
 1. When the applicant receives the email reminder that the application is about to expire (or has expired), they can login and click the **Renew** button.
+
    ![DACO renew button for an expired application](/assets/data-access/daco-renew-button.png)
-1. At this time, the application will be unlocked and you can make any changes (if necessary) to the currently approved application, agree to the Data Access Agreement and Appendices again, and re-sign and submit the renewal application.
+
+2. Upon clicking the **Renew** button, a new, separate draft renewal application is created. This draft is linked to the original application for reference only. The applicant must complete this new draft application for renewal.
+
+   - **Note:** While the original application remains in the APPROVED state some fields can be edited. Collaborators can be added or removed, and ethics documents can be appended.
+
    ![DACO renewal application](/assets/data-access/daco-renewal-application.png)
-1. The DACO Administrator will be notified that a renewal application is ready for review. If there are no issues and/or the application remains the same as the previous version, the project team should be quickly approved for an additional 2 years.
+
+3. Following submission, the DACO Administrator will be notified that a renewal application is ready for review. If there are no issues and/or the application remains consistent with the previous version, the project team should be quickly approved for an additional 2 years.
 
 ### Expired and Non-renewable
 
-A signed copy of the renewal application must be sent to the ICGC DACO via the online application system. If a renewal is not received within 90 days of the application's expiry date, it will be considered closed permanently.
+A signed copy of the renewal application must be sent to the ICGC DACO via the online application system. If a renewal is not submitted within 90 days of the original application's expiry date, the renewal will be permanently expired and you will need to submit a new application to regain access.
+
 ![DACO renewal period is up so application is non-renewable](/assets/data-access/daco-renewal-is-up.png)
 
 ## Closing an Application
 
-1. The application can also be closed by the Applicant or by a DACO Administrator at any time within the lifecycle of the application (before or after approval). A **Close Application** button is available from within the application header bar.
+1. The application can be closed by the Applicant or by a DACO Administrator at any time within the lifecycle of the application (before or after approval). A **Close Application** button is available from within the application header bar.
```

Comment:
not for this PR, but just noticed the language we're using here, referring to the `Applicant` - we may need to revisit where this is used in the context of the submitter/applicant (PI) confusion